### PR TITLE
copy_to_archive: copy bmap file

### PR DESCRIPTION
**Pull request by fhajredini**
Signed-off-by: Fisnik Hajredini <fhajredini@luxoft.com>

### CODE REVIEW

**Comment by sashko** [comment]
why don't I have `wic.bz2` image?

File: ci-scripts/copy_to_archive [hunk 1]
```diff
@@ -47,6 +48,7 @@ fi
 
 if [ -d ${IMAGE_DIR}/${ARP_ARCH} ]; then
     echo "Archiving ARP image to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
```
Comment (martin-ejdestig) [inline ci-scripts/copy_to_archive]: INTEL_ARCH -> ARP_ARCH

**Comment by fhajredini** [comment]
After discussing with Oleksi*, i think that including the bz2 is not worthy as it will only take more time to archive it and take more space. For now I will only add the bmap file.

The compressed image shall be added when a decision to modify the muxpi script is taken. In that way, we can remove the raw image, and only archive the compressed image.

**Comment by fhajredini** [comment]
ping @sashko @martin-ejdestig

**Pull request merged**